src/applications/personalization/dashboard-2/components/health-care/Appointments.jsx

```diff
@@ -28,63 +20,23 @@ export const Appointments = ({ appointments }) => {
     locationName = nextAppointment?.providerName;
   }
 
-  const cardDetails = {
-    ctaIcon: 'calendar',
-    ctaHref: '/health-care/schedule-view-va-appointments/appointments',
-    ctaAriaLabel: 'Manage all appointments',
-    ctaOnClick: recordDashboardClick('manage-all-appointments'),
-    ctaText: 'Manage all appointments',
-  };
-
-  // has an upcoming appointment in the next 30 days
-  if (hasFutureAppointments) {
```

Comment:
I was able to remove all this logic from this component since this component is now only responsible for showing scheduled appointments. If the user does not have anything scheduled, we don't render this component.